docs/nuget-org/nuget-org-faq.yml

```diff
@@ -60,33 +60,55 @@ sections:
           It is possible to share dependencies between multiple packages, making the total download size for consumers of your NuGet packages smaller.
           
           Dependencies are mostly static and never change. When fixing a bug in code, the dependencies may not need to be updated. If you bundle dependencies, you end up reshipping larger packages every time. By splitting NuGet packages into related dependencies, upgrades are much more fine-grained for consumers of your package.
-          
+
   - name: NuGet.org not accessible
     questions:
-      - question: Why can't I download packages from or upload packages to NuGet.org?
+      - question: What TLS version and cipher suites does NuGet.org support?
         answer: |
-          First, make sure you're using the latest versions of NuGet. If that version continues to fail, [contact support](https://www.nuget.org/policies/Contact) and provide additional connection troubleshooting information including:
-          
-          - The version of NuGet you're using
+          NuGet.org supports TLS 1.2 and the following cipher suites:
+          - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+          - TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+          - TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
```

Comment:
Would it be possible to link to some of the OS-specific guidance? Like "If you are using Windows XP or Windows Server 2003, TLS 1.2 support is possible with some extra steps..."

Reply:
This seems fine for now? Both these OS are deprecated anyway.

Reply:
Sure, sounds fine to me. I know that over support requests we often give more tailored guidance based on OS version.

Reply:
I decide not to list the details of the OS-specific guidance because there are already many official documents online talking about how to configure cipher suites on different systems. Once we list the supported cipher suites, customers should be able to find documents on how to configure it and it is not our job to document the detailed steps.

The cipher suites that CDN supported before are not common ones so that's why we put some instructions when addressing support tickets.